diagnostic.md

```diff
@@ -73,14 +88,27 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request PUT "http://example.com/donuts/$ID"
+  --header "Content-Type: application/json" \
+    --data '{
+        "name": "French Cruller",
+        "price": "$0.99"
+      }
+    }'
+
```

Comment:
a few things: 1) recall that the id number is not something we handle when making the curl request. It is managed by the database. This means that the URL to which we make the request doesn't not need to specify an id number. 2)  the data object is a "donut," but that's not denoted anywhere in your data object. 

Here's the solution:
```
curl --include --request "POST" "http://example.com/donuts" \
  --header "Content-Type: application/json" \
  --data $'{
    "donut": {
      "name": "French Cruller",
      "price": "$0.99"
    }
  }'
```